e2e/test/scenarios/joins/reproductions/29795-native-query-join-question-misses-metadata.cy.spec.js

```diff
@@ -0,0 +1,43 @@
+import {
+  restore,
+  visualize,
+  popover,
+  openOrdersTable,
+} from "e2e/support/helpers";
+
+describe("issue 29795", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it.skip("should allow join based on native query (metabase#29795)", () => {
+    const NATIVE_QUESTION = "native question";
```

Comment:
If you're not going to assert on the question's name, feel free to omit this.
There is a fallback string for the `name`.